### PR TITLE
Ignore CMAKE_CXX_COMPILER warning in HTML log

### PR DESCRIPTION
**Pull request by TimoWilken**
Should be disabled anyway, so remove the clause that looks for it.

Previously, we found the following messages by looking for lines of 79 asterisks:

```
*******************************************************************************
*----------------------------------- ERROR -----------------------------------*
* The variable 'CMAKE_CXX_COMPILER' should only be set by the cmake toolchain,
* either by calling 'cmake -DCMAKE_CXX_COMPILER="/opt/rocm/hip/bin/hipcc"' or
* set in a toolchain file and added with 
* 'cmake -DCMAKE_TOOLCHAIN_FILE=<toolchain-file>'.
*-----------------------------------------------------------------------------*
*******************************************************************************
```

### CODE REVIEW

**Comment by ktf** [comment]
Shouldn't we fix the error in this particular case? It's not a warning. What is actually happening?

**Comment by davidrohr** [comment]
@ktf : It is not an error. AMD complains about something that is totally valid in CMake. There is an option to disable the error in the AMD CMake, but this just changes it from a CMake fatal to CMake message, so it is still printed out. So it is just a message text string that says error, which we were grepping for. I have reported the false behavior to AMD, but will take a while until it is fixed in a release. On top of that, this check should not be executed on our CMake code anyway, but only within the AMD CMake internally. So this is a second bug in the AMD CMake that the check triggers at all in our code, which I have also reported to them, but again will take a while to be fixed.
Bottom line, let's just not grep for it :)
